build.earth

```diff
@@ -0,0 +1,58 @@
+VERSION 0.8
+
+
+SRC:
+	FUNCTION
+	COPY --dir scripts stdeb test_data \
+		test*.sh *.txt *.py *.cfg *.toml *.rst \
+		MANIFEST.in \
+		/src/stdeb
+	WORKDIR /src/stdeb
+
+BUILD:
+	FUNCTION
+	ENV DEBIAN_FRONTEND=noninteractive
+	# Build deps
+	RUN apt-get update; apt-get install -y debhelper dh-python python3-all python3-pip
```

Comment:
For performance reasons maybe use a single `apt-get update and apt-get install`  invocation and if you want to keep the dependencies separated use env vars?